### ISSUE TITLE
Exception in thread "main" java.util.zip.ZipException: error in opening zip file

### ISSUE THREAD

**Issue by vincentcox-work** [WebGoat/WebGoat#337]
I did the following:
```
sudo apt-get install default-jre
sudo apt-get install default-jdk
wget https://github.com/WebGoat/WebGoat/releases/download/7.1/webgoat-container-7.1-exec.jar
sudo chmod +x webgoat-container-7.1-exec.jar
java -jar webgoat-container-7.1-exec.jar
```
I get the following error.
`
Exception in thread "main" java.util.zip.ZipException: error in opening zip file
        at java.util.zip.ZipFile.open(Native Method)
        at java.util.zip.ZipFile.<init>(ZipFile.java:219)
        at java.util.zip.ZipFile.<init>(ZipFile.java:149)
        at java.util.jar.JarFile.<init>(JarFile.java:166)
        at java.util.jar.JarFile.<init>(JarFile.java:103)
        at sun.net.www.protocol.jar.URLJarFile.<init>(URLJarFile.java:93)
        at sun.net.www.protocol.jar.URLJarFile.getJarFile(URLJarFile.java:69)
        at sun.net.www.protocol.jar.JarFileFactory.get(JarFileFactory.java:84)
        at sun.net.www.protocol.jar.JarURLConnection.connect(JarURLConnection.java:122)
        at sun.net.www.protocol.jar.JarURLConnection.getInputStream(JarURLConnection.java:150)
        at org.apache.tomcat.maven.runner.Tomcat7Runner.getContextXml(Tomcat7Runner.java:510)
        at org.apache.tomcat.maven.runner.Tomcat7Runner.run(Tomcat7Runner.java:414)
        at org.apache.tomcat.maven.runner.Tomcat7RunnerCli.main(Tomcat7RunnerCli.java:244)
`
Any idea's?

**Comment by nbaars** [comment]
@vincentcox-ordina By default the executable jar creates a hidden directory to extract the zip file, can you check whether the user is allowed to create a directory within the location you start WebGoat?

**Comment by nbaars** [comment]
There is a Docker Hub version of WebGoat available for WebGoat 7.1 as well (https://hub.docker.com/r/webgoat/webgoat-7.1/)
